Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -286,20 +286,17 @@ def populate(self, issue, member=None):
             issue._references = []
             refs = set()
 
-            # Attempt to match radar importer first
-            if self.radar_importer:
-                for comment in (issue.comments or []):
-                    if not comment:
-                        continue
-                    if comment.user != self.radar_importer:
-                        continue
-                    candidate = GenericTracker.from_string(comment.content)
-                    if not candidate or candidate.link in refs or (isinstance(type(candidate.tracker), type(issue.tracker)) and candidate.id == issue.id):
-                        continue
-                    issue._references.append(candidate)
-                    refs.add(candidate.link)
+            # In reverse chronological order, prioritize comments with only a bug url, then all comments, then description
```

Comment:
"only a bug URL" is a notable change from the Radar importer, which always just goes by comments from newest-to-oldest (and the description in Bugzilla is just comment 0!)

Reply:
I guess there's no reason not to just post the importer logic here:

```python
import re

RADAR_LINK_REGEX = r"^<?(rdar|radr|radar)://(problem/)?(?P<radar_id>\d+)>?.*$"

for comment in reversed(list(comments_for_bug(bugzilla_id))):
    match = re.search(RADAR_LINK_REGEX, comment["text"], re.MULTILINE)
    if match:
        radar_id = match.group("radar_id")
```

Notably this differs insofar as it _requires_ the link to appear at the start of a line, though it allows anything following it.

Reply:
Yeah I think we should follow WKBI. I've updated the regex and logic to reflect this.